Makefile

```diff
@@ -1,3 +1,24 @@
+MAIN=dp-reporter-client
+
+BUILD=build
+BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
+BIN_DIR?=.
+
+PHONY: all
+all: audit test build
+
+PHONY: audit
+audit:
+	go list -m all | nancy sleuth
+
+PHONY: build
+build:
+	@mkdir -p $(BUILD_ARCH)/$(BIN_DIR)
+	go build $(LDFLAGS) -o $(BUILD_ARCH)/$(BIN_DIR)/$(MAIN)
```

Comment:
struggling with this line:
- `LDFLAGS` is not set, so can probably be removed
- in fact, this doesn't build anything for me, so wondering what `make build` should do in general (if anything) - will speak to you about it [note2]